### DISCUSSION TITLE
Deploy a job that gets triggered at a specific time

### DISCUSSION THREAD

**Discussion by IgnisDa** [apalis-dev/apalis#138] [Q&A] [answered]
Is it possible to deploy a job that gets triggered at a specific time? For eg: 2 hrs from now (duration), or at 24-08-2023 10:05:30 (specific time).

**Comment by geofmureithi** [comment, marked as answer]
Storage's offer a schedule method which is similar to push.
https://docs.rs/apalis-core/0.4.4/apalis_core/storage/trait.Storage.html#tymethod.schedule
```rust
fn schedule(
    &mut self,
    job: Self::Output,
    on: DateTime
)
```
